app/metrics.yaml

```diff
@@ -205,6 +205,128 @@ events:
       - fenix-core@mozilla.com
     expires: "2020-09-01"
 
+onboarding:
+  fxa_auto_signin:
+    type: event
+    description:
+      The onboarding automatic sign in card was tapped.
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/10824
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/issues/10824
+    notification_emails:
+      - fenix-core@mozilla.com
+      - erichards@mozilla.com
+    expires: "2020-09-01"
+  fxa_manual_signin:
+    type: event
+    description:
+      The onboarding manual sign in card was tapped.
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/10824
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/issues/10824
+    notification_emails:
+      - fenix-core@mozilla.com
+      - erichards@mozilla.com
+    expires: "2020-09-01"
+  privacy_notice:
+    type: event
+    description:
+      The onboarding privacy notice card was tapped.
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/10824
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/issues/10824
+    notification_emails:
+      - fenix-core@mozilla.com
+      - erichards@mozilla.com
+    expires: "2020-09-01"
+  pref_toggled_private_browsing:
+    type: event
+    description:
+      The private browsing preference was selected from the onboarding card.
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/10824
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/issues/10824
+    notification_emails:
+      - fenix-core@mozilla.com
+      - erichards@mozilla.com
+    expires: "2020-09-01"
+  pref_toggled_toolbar_position:
+    type: event
+    description:
+      The toolbar position preference was chosen from the onboarding card.
```

Comment:
Let's make sure to include the default state for all of these prefs getting toggled 😄